packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -28,7 +28,6 @@ contract TaikoL1 is TaikoL1Base {
             blockMaxVerificationsPerTx: 10,
             blockMaxGasLimit: 6_000_000,
             blockFeeBaseGas: 20_000,
-            blockMaxTransactions: 79,
```

Comment:
There's a separate limitation inside the circuits because transactions are handled separately, so there is for example a limited amount of signatures we can check. But yeah we can check this limitation inside the circuits instead as long as that limitation is there.